webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const webpack = require('webpack');
+
+const debug = process.env.DEBUG;
+
+module.exports = {
+   module: {
+      rules: [
+         {
+            test: /\.html$/,
+            use: 'raw-loader'
+         }
+      ]
+   },
+   resolve: {
+      modules: ['public/js', 'views/current', 'node_modules']
+   },
+   plugins: [
+      new webpack.ProvidePlugin({
+        $: "jquery",
```

Comment:
Is this necessary? It looks like you have `import $ from 'jquery'` in most files.

Reply:
Ah yeah, so this is necessary to make Bootstrap not die when bundling. I didn't realize this until after I put `import $ from 'jquery'` in our files that needed it, so I suppose I don't really need all those explicit jquery imports.

Is it common to just add utility libraries to ProvidePlugin so that there's not an extra `import _` or `import $` in every file, or is it better to be explicit?

Reply:
That makes sense. Thanks for explaining.

I think it's better to keep it explicit in our own source files.

Reply:
Cool! I think that sounds good. If we ever change out bootstrap for something else we could remove the provider plugin and then we won't have to re-add imports to all of our files.